core/src/main/java/org/infinispan/interceptors/AsyncInterceptorChain.java

```diff
@@ -8,24 +8,24 @@
 import java.util.concurrent.CompletableFuture;
 
 /**
- * Interceptor chain using {@link SequentialInterceptor}s.
+ * Interceptor chain using {@link AsyncInterceptor}s.
  *
  * Experimental: The ability to modify the interceptors at runtime may be removed in future versions.
  *
  * @author Dan Berindei
  * @since 9.0
  */
 @Experimental
-public interface SequentialInterceptorChain {
-   List<SequentialInterceptor> getInterceptors();
+public interface AsyncInterceptorChain {
+   List<AsyncInterceptor> getInterceptors();
```

Comment:
javadoc missing. it would be good to know if the return list is immutable or not.